### ISSUE TITLE
Default Value "0" for a number field does not work

### ISSUE THREAD

**Issue by jonaswouters** [verbb/formie#1283]
### Describe the bug

When you have a number field on your form, and want to use “0” as your default value, it will not be rendered on the form. Instead, the form field will be empty.
When you enter a number higher than 0, it does work. 

### Steps to reproduce

1. Add a number field to your form
2. Set the default value to: 0
3. View form on the website

### Form settings

- Multi-page form: Yes
- Submission Method: Ajax or Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

4.3.8

### Plugin version

2.0.21

### Multi-site?

No

### Additional context

Not requiring the field would be an option, but sometimes clients want it this way.

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.21",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.22](https://github.com/verbb/formie/releases/tag/2.0.22)
